lib-rgbpanel/include/rgbpanel.h

```diff
@@ -31,6 +31,10 @@
 
 #include "rgbpanelconst.h"
 
+#include "../../lib-device/src/font_cp437.h"
+#include "../../lib-device/src/font_5x8.h"
```

Comment:
font_5x8.h is currently a lib-rgbpanel only solution. It is better move it to lib-rgbpanel

Reply:
https://github.com/hippyau/rpidmx512/commit/cf0c918b65a35cce53ba251e2f3331f68dbdfb42

Reply:
Hey Arjan, 
   Wow, I just lost 4 hours to a bug related to this early move.   I moved the font to lib-rgbpanel,  I thought would be easy, so didn't test straight away, when I made a bunch of changes and got weird stuff on the display...      well, i've tracked back and simply moving the font files .c and .h from lib-device to lib-rgbpanel has screwed me over :) 

I dont really know why,  I am thinking the declaration as extern when it's only been used inside lib-rgbpanel, or maybe not declared right some other way, but I don't think so...?  no idea why location is an issue...

It's like when the font is in lib-rgbpanel,  display looks like the pointer is off or something when it comes time to actually PutChar.    

I am stumped, so moved back into lib-device and it's perfect?!?!?   I check by reverting, and it's broken again.  

I wonder, wtf, no solid ideas...  so I leave in lib-device for now :)

Maybe because I had to put the .h file in include and the .c file in src,  where in lib-device that are both in src directory, but yeah nah.